_includes/header.html

```diff
@@ -1,14 +1,14 @@
 <header class="site-header">
   <div class="container">
-    <h1 class="site-title">
+    <h3 class="site-title">
       <a href="{{ site.baseurl }}/" rel="home">
```

Comment:
I think from a semantic perspective we want to leave this as h1, can we adjust the css to resize?